docs/General/Presentations/index.md

```diff
@@ -1,7 +1,42 @@
 # Presentation Standards
 
-TODO: #35 Add Presentation Standards
+## Talk Standards
+
+**Key Slides**
```

Comment:
Might be nicer to use markdown subheadings instead of bold text? This would apply to all the subheadings on the page.

```suggestion
### Key Slides
```